docker/settings/services.env

```diff
@@ -4,4 +4,4 @@ POSTGRES_DB=test
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=testPass
 POSTGRES_HOST=localhost
-POSTGRES_PORT=5435
+POSTGRES_PORT=5445
```

Comment:
Any especial reason for the apparently random change of port number? I don't see 5445 in https://liv-it.atlassian.net/wiki/spaces/UP/pages/72679452/Development+Ports+Mapping.

Reply:
It was clashing with something from local deockerized dev instance of the LTI. I added 5445 to the wiki you referenced (I wasn't aware of it before)